src/features.F90

```diff
@@ -42,8 +37,6 @@ function get_xtb_feature(feature) result(has_feature)
    select case(feature)
    case("tblite")
       has_feature = tblite_support
-   case("cpcmx")
-      has_feature = cpcmx_support
```

Comment:
Let's keep this and always return false here

Reply:
Ok. Give me couple minutes ;-)